assignment_3/Stringoperations/src/StringOperations.java

```diff
@@ -0,0 +1,201 @@
+import java.util.Scanner;
+
+
+public class StringOperations
+{
+	/**
+	 * 
+	 * @param first string s1
+	 * @param first string s2
+	 * @return 1 if both strings are equals and 0 if unequal
+	 */
+	
+	public int Comparison(String s1,String s2)
+	{
+		char c1[]= new char[s1.length()];
+		char c2[]= new char[s2.length()];
+		
+		c1=s1.toCharArray();
+		c2=s2.toCharArray();
+		
+			if(c1.length==c2.length)
+			{
+				int flag=0;
+				for(int i=0;i<c1.length;i++)
+				{
+					if(c1[i]!=c2[i])
+					{
+						flag=1;
+						return 0;
+					}
+				}
+			if(flag==0)
+			{
+				return 1;
+			}
+			}
+			else
+			{
+				return 0;
+			}
+			
+			return 0;
+	}
+	
+	/**
+	 * 
+	 * @param String s1 which is to be reversed
+	 * @return reversed string
+	 */
+	
+	public String string_reversal(String s1)
```

Comment:
Method name should be in camel case and variable names should be meaningful
for ex: stringReversal(String stringToReverse)